docs/user-guide/commands/argocd_proj_remove-destination.md

```diff
@@ -5,7 +5,7 @@
 Remove project destination
 
 ```
-argocd proj remove-destination PROJECT SERVER NAMESPACE [flags]
+argocd proj remove-destination PROJECT SERVER NAMESPACE SERVICEACCOUNT [flags]
```

Comment:
This change is not required. As we have a separate object DestinationServiceAccount under AppProject